spec/latest/json-ld/index.html

```diff
@@ -3964,6 +4531,11 @@ <h2>Changes since 1.0 Recommendation of 16 January 2014</h2>
       a <code>@prefix</code> member with the value <a>true</a>. The 1.0 algorithm has
       been updated to only consider terms that map to a value that ends with a URI
       <a data-cite="RFC3986#section-2.2">gen-delim</a> character.</li>
+    <li>Values of properties where the associated <a>term definition</a>
+      has <code>@container</code> set to <code>@graph</code> are interpreted as
+      <a>implicitly named graphs</a>, where the associated graph name is either
+      assigned from a new <a>blank node identifier</a>, or defined using
+      <code>@graphId</code>.</li>
```

Comment:
@gkellogg Was this supposed to be removed?

Reply:
No, this remains, and is what is used by verifiable credentials. We may need to add more about graph id and index containers, though.

What we removed was anything about graphId.